mmv1/third_party/terraform/services/composer/resource_composer_environment_test.go.tmpl

```diff
@@ -22,27 +22,27 @@ const testComposerNetworkPrefix = "tf-test-composer-net"
 const testComposerBucketPrefix = "tf-test-composer-bucket"
 const testComposerNetworkAttachmentPrefix = "tf-test-composer-nta"
 
-func bootstrapComposerServiceAgents() {
-	acctest.BootstrapIamMembers([]acctest.IamMember{
+func bootstrapComposerServiceAgents(t *testing.T) {
+	acctest.BootstrapIamMembers(t, []acctest.IamMember{
 		{
-			member: "serviceAccount:service-{project_number}@cloudcomposer-accounts.iam.gserviceaccount.com",
-			role:   "roles/cloudkms.cryptoKeyEncrypterDecrypter",
+			Member: "serviceAccount:service-{project_number}@cloudcomposer-accounts.iam.gserviceaccount.com",
+			Role:   "roles/cloudkms.cryptoKeyEncrypterDecrypter",
 		},
 		{
-			member: "serviceAccount:service-{project_number}@compute-system.iam.gserviceaccount.com",
-			role:   "roles/cloudkms.cryptoKeyEncrypterDecrypter",
+			Member: "serviceAccount:service-{project_number}@compute-system.iam.gserviceaccount.com",
+			Role:   "roles/cloudkms.cryptoKeyEncrypterDecrypter",
 		},
 		{
-			member: "serviceAccount:service-{project_number}@container-engine-robot.iam.gserviceaccount.com",
-			role:   "roles/cloudkms.cryptoKeyEncrypterDecrypter",
+			Member: "serviceAccount:service-{project_number}@container-engine-robot.iam.gserviceaccount.com",
```

Comment:
are there ever cases where you want to allow a different TLD? Would it work to end at `foo` or `foo.iam`, or is that too magical?

Reply:
I'd prefer not to be at all magical for the bootstrapping... I know it's more verbose & a bit duplicative but it's much more clear about what's happening. We can always consider adding shortcuts for building the members later (and if we did that, we could make those available as provider functions as well).

Reply:
I've had too many times (in various situations) where I have to debug something and end up digging through like twelve layers of "helpers" to figure out what is actually happening. 😂

Reply:
Totally makes sense!

And with the various wrappers / helpers also (as discussed in the other tickets), it might be nice to flatten them a bit... right now, there's a helper for if you want one member with multiple roles, another if you want multiple members with one role, and so on....